docs/skopeo-sync.1.md

```diff
@@ -0,0 +1,131 @@
+% skopeo-sync(1)
+
+## NAME
+skopeo\-sync - Synchronize images between container registries and local directories.
+
+
+## SYNOPSIS
+**skopeo sync** [**sync-options**] --src _source-location_ --dest _destination-location_ _source_ _destination_
```

Comment:
I'd remove "[**sync-options**]"  Most of our pages don't add that, it's implied that the options go there.